lib/formatters.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const Os = require('os');
+const Stringify = require('fast-safe-stringify');
+
+const influxInt = (value) => {
+    if (Number.isInteger(Number(value))) {
```

Comment:
@sfabriece I tried to maintain the fix you added to Int, from `line-protocol.js`.  Please review.